test/tests/float.py

```diff
@@ -88,3 +88,16 @@ class F2(float):
     float(-l)
 except OverflowError as e:
     print e.message
+
+all_args = [None, 1, 1L, -1, -1L, 2.0, 0.5, 0]
+for lhs in all_args:
+    for rhs in all_args:
+        for mod in all_args:
+            try:
+                pow(lhs, rhs, mod)
+            except ValueError as e:
+                print e.message
+            except TypeError as e:
+                print e.message
+            except ZeroDivisionError as e:
+                print e.message
```

Comment:
Here are some more tests that currently don't work:
`pow(1<<30, 1<<30, 127)`
`pow(1L<<30, 1L<<30, 127)`

It's probably safe to leave those as unimplemented right now, but currently those will cause us to run out of memory.